templates/adherent/overview.html.twig

```diff
@@ -0,0 +1,28 @@
+{% extends 'adherent/layout.twig' %}
+
+{% block adherent_profile_form %}
+
+    <h2>Mon compte</h2>
+
+    <p>
```

Comment:
`p` tags should not contain block level tags.

see https://www.w3.org/TR/html401/struct/text.html#h-9.3.1

> The P element represents a paragraph. It cannot contain block-level elements (including P itself).

Reply:
😳 I wasn't thinking. Will send a PR back shortly.